style.css

```diff
@@ -0,0 +1,79 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+  color: white;
+}
+
+body {
+  background-image:
+    linear-gradient(rgba(88, 89, 146, 0.7), rgba(88, 89, 146, 0.7)),
+    url(./images/banner.jpeg);
+  background-position: center;
+  background-size: cover;
+
+  /* position: relative; */
```

Comment:
Please do not leave unused code as a comment in your codebase. Unused code as comments makes your code not clean.